web/carousel/src/screens/index.js

```diff
@@ -10,6 +10,31 @@ import UserInfo from "../components/Context/UserInfo";
 import Header from "../components/Header/Header";
 import NotFound from "./NotFound";
 
+
+function PrivateRoute ({component: Component, authed, ...rest}) {
+  console.log("authed: ", localStorage.getItem("login"));
+  return (
+    <Route
+      {...rest}
+      render={(props) => localStorage.getItem("login") === "true"
+        ? <Component {...props} />
+        : <Redirect to={{pathname: '/login', state: {from: props.location}}} />}
+    />
+  )
+}
+
+function HomeRoute ({component: Component, authed, ...rest}) {
+  return (
+    <Route
+
+      {...rest}
+      render={(props) => authed === true
+        ? <Component {...props} />
+        : <Redirect to={{pathname: '/account', state: {from: props.location}}} />}
+    />
+  )
+}
+
```

Comment:
If we won't use the HomeRoute function, we can remove it. It is a little confusing.